src/cruise_control/scripts/cluster_values.txt

```diff
@@ -0,0 +1,2 @@
+35
```

Comment:
Would it make sense to centralize our magic number constants using something like environment variables? We could keep them all in one file which makes it much easier to keep track of.

Reply:
I think Babak even requested this if I am remembering correctly. Not sure if we need to make this a requirement for this specific ticket, but it should be done at some point. There are a decent amount for lka.

Reply:
We can do this in another PR